python/tvm/contrib/cutlass/build.py

```diff
@@ -35,6 +35,8 @@ def has_cutlass():
     """Returns true if the CUTLASS custom codegen is available"""
     return tvm.get_global_func("relay.ext.cutlass.create_c_source_module", True) is not None
 
+def ddd():
+    print("testing")
 
 def _get_cutlass_path():
```

Comment:
Add more tests to ensure that the code is working correctly and to prevent future regressions [medium]